oc.spec

```diff
@@ -116,11 +132,11 @@ done
 %dir %{_datadir}/%{name}/macosx/
 %dir %{_datadir}/%{name}/windows/
 %{_datadir}/%{name}/linux/oc
-#%{_datadir}/%{name}/linux/kubectl
```

Comment:
I guess we didn't have the symlink at the time, I don't see other reason to keep these off

Reply:
looks like we still need those, let's address it separately